RedHatInsights/vulnerability-ui
# Optimize select all items in CVE detail and System detail tables

Resolves [VULN-2364](https://issues.redhat.com/browse/VULN-2364).

**How to test:**
Just verify select page/select all works as expected in Systems page, CVE detail page and System detail page.
